Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y git
+
+COPY . .
```

Comment:
Do not do `COPY . .` as that has the risk of copying sensitive files into the container which is then later published.